README.md

```diff
@@ -0,0 +1,35 @@
+# Mouse Recorder
+###A cheap, modular, behavior recording program
+![](http://s16.postimg.org/yek83sdv9/Mouserecord_logo.jpg 2)
```

Comment:
Also, the image should probably be moved into the repo. Can you add it and commit to the repo? Once added we should replace the URL with a relative path to the file. ( https://github.com/blog/1395-relative-links-in-markup-files )